rest-api-model-ql/src/main/kotlin/org/modelix/sample/restapimodelql/Paths.kt

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Is this generated during the gradle build? Or why is this committed if there is also a gradle plugin for the openapi generator?

Reply:
the model-ql sample folder is based off the json-bulk one. there the paths were generated once but due to openapi kotlin generator issues the decision was to check in the Paths rather than re-generating every time. same reasoning applies for the new example.

Reply:
ah, that was the one shot generation.